packages/replay-internal/src/types/replayFrame.ts

```diff
@@ -163,10 +163,25 @@ interface ReplayHistoryFrame extends ReplayBaseSpanFrame {
 }
 
 interface ReplayLargestContentfulPaintFrame extends ReplayBaseSpanFrame {
-  data: LargestContentfulPaintData;
+  data: WebVitalData;
   op: 'largest-contentful-paint';
 }
 
+interface ReplayCumulativeLayoutShiftFrame extends ReplayBaseSpanFrame {
+  data: WebVitalData;
+  op: 'cumulative-layout-shift';
+}
+
+interface ReplayFirstInputDelayFrame extends ReplayBaseSpanFrame {
+  data: WebVitalData;
+  op: 'first-input-delay';
+}
+
+interface ReplayInteractionToNextPaintFrame extends ReplayBaseSpanFrame {
+  data: WebVitalData;
+  op: 'interaction-to-next-paint';
+}
```

Comment:
I would condense this down to a single type since the data is all the same:

```suggestion
interface ReplayWebVitalFrame extends ReplayBaseSpanFrame {
  data: WebVitalData;
  op: 'cumulative-layout-shift' | 'first-input-delay' | 'interaction-to-next-paint';
}
```